source/limitations.txt

```diff
@@ -0,0 +1,96 @@
+.. _entity-framework-limitations:
+
+===========
+Limitations
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta:: 
+   :keywords: .NET, support
+
+Overview
+--------
+
+On this page, you can find a list of MongoDB and {+framework+} features that
+are not supported by the {+provider-long+}. Because the provider is in active
+development, some features listed on this page might be considered for future
+releases based on customer demand.
```

Comment:
S: since we dont at this point exactly know how features are prioritized, this might be an unnecessary level of info

```suggestion
development, some features listed on this page might be considered for future
releases.
```

Reply:
For this page, the engineers asked us to specify that consideration of these features would be based on customer demand.